server/server.ts

```diff
@@ -14,6 +16,84 @@ const PORT = process.env.PORT || 8000;
 const app = express();
 app.use(express.json());
 app.use(express.static('../client'));
+app.use(require('morgan')('combined'));
+
+// From Passport example -- not sure if all these dependencies are necessary
+app.use(require('cookie-parser')());
+app.use(require('body-parser').urlencoded({ extended: true }));
+app.use(require('express-session')({
+  secret: 'keyboard cat', // TODO ?
+  resave: true,
+  saveUninitialized: true,
+}));
+
+app.use(passport.initialize());
+app.use(passport.session());
+app.enable("trust proxy");
```

Comment:
❓ About "trust proxy":

I'm still not super sure if this is the right thing to do! One alternative I could've used: Change `/return` to an absolute URL with HTTPS

https://stackoverflow.com/questions/20739744/passportjs-callback-switch-between-http-and-https/20848306
https://stackoverflow.com/questions/23413401/what-does-trust-proxy-actually-do-in-express-js-and-do-i-need-to-use-it/42765157